include/parser.php

```diff
@@ -749,17 +749,17 @@ function ($match) { return handle_list_tag($match[2], $match[1]); },
 		);
 	}
 
-	$content = preg_replace('#\s*\[\*\](.*?)\[/\*\]\s*#s', '<li><p>$1</p></li>', pun_trim($content));
+	$content = preg_replace('#\s*\[\*\](.*?)\[/\*\]\s*#s', '<li>$1</li>', pun_trim($content));
```

Comment:
Why is the paragraph tag being removed here? That seems like an unwanted change to the current markup?

Reply:
It was made to have correct impact. But I think I need to revise this!

Still you can merge it without `p` which is the right thing to do. Later I will share some workaround if needed

Reply:
Having `p` in the `li` makes issue. If it is necessary to have paragraph in list, let me know so I make required changes.